neural_compressor/model/onnx_model.py

```diff
@@ -471,6 +471,11 @@ def topological_sort(self, enable_subgraph=False):
 
     def get_nodes_chain(self, start_node, stop_node, result_chain=[]):
         """Get nodes chain with given start node and stop node."""
+        from collections import deque
+        if not all([isinstance(node, str) for node in start_node]):
```

Comment:
'not all([isinstance(node, str) for node in start_node])' doesn't mean all objects in start_node is onnx node object